src/script/user/UserRepository.ts

```diff
@@ -249,6 +252,12 @@ export class UserRepository {
       }
     });
 
+    // Map self user's availability status
+    const selfUserAvailability = dbUsers.find(user => user.id === selfUser.id);
```

Comment:
This one shouldn't be called availability, it seems to be user's db record, maybe just `selfUser` wdyt?

Reply:
Humm that's indeed good feedback. I named it this way because I only use the availability from it, but you're right, it's misleading 👌